soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -939,10 +949,9 @@ void CheckTrackerWindow::DrawElement() {
             previousShowHidden = showHidden;
             doAreaScroll = true;
         }
-        if (!showHidden && (
-            hideComplete && thisAreaFullyChecked || 
-            hideIncomplete && !thisAreaFullyChecked
-        )) {
+        if (shouldHideArea(checkSearch, checksByArea, rcArea) ||
+            (!showHidden && ( hideComplete && thisAreaFullyChecked || hideIncomplete && !thisAreaFullyChecked))
```

Comment:
I know this wasn't your fault, but could you encapsulate the two parts around the || here in parentheses? That's how it should have been, might as well get it done now. Should be `(!showHidden && ((hideComplete && thisAreaFullyChecked) || (hideIncomplete && !thisAreaFullyChecked)))`